modules/ROOT/pages/_partials/upgrade-tool.adoc

```diff
@@ -0,0 +1,61 @@
+//This content is reused in:
+//mule-upgrade-tool.adoc
+
+// Before You Begin
+// tag::BeforeYouBegin[]
+
+* A currently operational Mule instance running version 4.1.1 or later, in _stopped_ status to prepare for the upgrade.
++
+When updating your current Mule instance to a version between 4.1.1 and 4.1.4, use any patch update released after January 20, 2022. If the running version is not one of the previously mentioned, an error message appears stating that the Mule version cannot be upgraded unless an upgrade is made.
+* The Mule upgrade tool inside your current Mule instance's `$MULE_HOME/tools` directory, that is, your old or existing Mule instance. 
++
+To download the tool, log in to https://help.mulesoft.com/s/[Help Center^]. Under the Support menu, select *Downloads > Mule Runtime*.
```

Comment:
```suggestion
To download the latest version of the tool (recommended), log in to https://help.mulesoft.com/s/[Help Center^]. Under the Support menu, select *Downloads > Mule Runtime*.
```

Reply:
not sure about "(recommended)" but I am almost certain Julio said that made sense because it incorporates the latest bug fixes and perhaps other enhancements.

Reply:
Will leave the "(recommended)" and have it checked by Julio.